eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/CompositeFastList.java

```diff
@@ -238,7 +239,7 @@ public boolean addAll(Collection<? extends E> collection)
             return false;
         }
         Collection<? extends E> collectionToAdd =
-                collection instanceof FastList ? collection : FastList.newList(collection);
+                collection instanceof FastList ? collection : Lists.mutable.withAll(collection);
```

Comment:
I think this needs to be left as `FastList` as the implementation must be a `FastList`.

Reply:
Discarded this change.